deploy/crds/sriovnetwork_v1_sriovnetwork_crd.yaml

```diff
@@ -26,22 +26,30 @@ spec:
         metadata:
           type: object
         spec:
+          description: Specification describing how a NetworkAttachmentDefinition
```

Comment:
Shall we also change the file under `manifests/4.2/manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml`?
before this PR, they have same content.

Reply:
Yes. Will do.